### DISCUSSION TITLE
PulseVolume/Volume widget didn't work normally

### DISCUSSION THREAD

**Discussion by AbdelrhmanUZaki** [qtile/qtile#4748] [Q&A] [answered]
PulseVolume widget works normal until I refresh qtile `mod ctrl r`, its value is set to 0% in my bar, there is still control to raise, lower or mute volume, but in all cases, the widget show 0%
why is that happens? it was working fine

**Comment by elParaguayo** [comment]
Check the logs and see if there's an error message.

**Comment by AbdelrhmanUZaki** [comment, marked as answer]
Solved!

Run xrandr command in any other startup file like `.bash_profile` or something like that, not in autostart.sh qtile's file
